examples/dask/special-topics-rolling-average.ipynb

```diff
@@ -0,0 +1,179 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7df4bb88-37cc-4b8a-bb9d-b476102b133a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import dask.dataframe as dd\n",
+    "\n",
+    "taxi = dd.read_csv(\n",
+    "    \"s3://nyc-tlc/trip data/yellow_tripdata_2019-01.csv\",\n",
+    "    parse_dates=[\"tpep_pickup_datetime\", \"tpep_dropoff_datetime\"],\n",
+    "    storage_options={\"anon\": True},\n",
+    ").sample(frac=0.1, replace=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a2886561-f4be-481d-95c1-82997a609ae9",
+   "metadata": {},
+   "source": [
+    "For dataframe we are calling function `set_index` to set `tpep_pickup_datetime` columns as the index. This will sort data by index within and across partitions. At this point, data and labels are lazy collections. They won’t be read into the workers' memory until some other computation asks for them. Hence, before we pass this data to other tasks we will call `presist()`. This will ensure that loading of data is run only once."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f3baefab-6733-4b41-abc0-63a89d645fe8",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from dask.distributed import wait\n",
+    "\n",
+    "taxi = taxi.set_index(\"tpep_pickup_datetime\")\n",
+    "taxi = taxi.persist()\n",
+    "_ = wait(taxi)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "69b608b5-7289-4983-b619-417b47b027ce",
+   "metadata": {},
+   "source": [
+    "In code below we get the 30-day rolling average of base fare prices ie. on column `fare_amount` and convert the resulting series to DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6d6f4e2a-1eda-429e-be99-6e268cac24b1",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "rolling_fares = taxi.fare_amount.rolling(\"30D\").mean()\n",
+    "rolling_fares_df = rolling_fares.to_frame(name=\"fare_amount_rolled\")\n",
+    "type(rolling_fares_df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d65607a-5629-434a-a876-b5fda59334c4",
+   "metadata": {},
+   "source": [
+    "We then join the original dataframe `taxi` with dataframe `rolling_fares_df`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "56aa5203-6f48-4cf3-9a5a-3630569b1204",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "taxi_new = taxi.join(rolling_fares_df, how=\"outer\")\n",
+    "taxi_new"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa9e4e3c-3bba-49ce-9f90-17f775701ae7",
+   "metadata": {},
+   "source": [
+    "On running the cell above, you will see that now we have a new column `fare_amount_rolled` added to dataframe. "
```

Comment:
Can you, either via a screenshot hosted on S3 or in a markdown table, show what like 5 rows of the output look like? It would be good to just see that it works without having to actually run the code.